### PR TITLE
[Draft] New stuff of One Star

### PR DESCRIPTION
**Pull request by TorinoFermic**
Things left missing : 
- Masks
- Ozk hood

### CODE REVIEW

**Comment by Kurgis** [comment]
There's already problems, but considering this a draft I won't review them. I do question the "Military radiation suit" and "russian military gas mask" objects though, they are done out in One Star colors, and it seems odd that they are described this way. Shazbot has already made the outfit in a teal color, which would better fit a not One Star faction.

**Comment by TorinoFermic** [comment]
Well, it came from a file named OneStar dmi. Not sure why this is that but well I will fix as I progress.

**Comment by shazbot194** [comment]
If you need the teal ones, or any other color, I would be happy to provide it, or make you a rainbow of OZK-esk suits.

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -106,3 +106,11 @@
 	desc = "Twoooo!"
 	icon_state = "owl"
 	body_parts_covered = HEAD|FACE|EYES
+
+/obj/item/clothing/mask/gas/ozk_mask
+	name = "russian military gas mask"
+	desc = "Russian military gas mask, dosen't come with vodka blyat !"
```
Comment (RandomColonist) [inline code/modules/clothing/masks/gasmask.dm]: Remove space after "blyat", please. Also you can add coma before blyat too.

File: code/modules/clothing/suits/utility.dm [hunk 2]
```diff
@@ -113,3 +113,21 @@
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 60, rad = 100)
 	flags_inv = HIDEJUMPSUIT|HIDETAIL
 	price_tag = 100
+
+/*
+*  Ozk Suit
+*/
+/obj/item/clothing/suit/radiation
+	name = "Military radiation suit"
```
Comment (RandomColonist) [inline code/modules/clothing/suits/utility.dm]: Why capital letter?

**Comment by TorinoFermic** [comment]
New pr will be opened because too many conflicting DMIs

**Pull request closed without merging**